quantum.bst

```diff
@@ -80,14 +80,13 @@ STRINGS { m }
 FUNCTION {my.warning}
 { 'm :=
   warnings.are.errors
-    { "\ReferToPDFOutput" write$
```

Comment:
@dwierichs I would like to replace this command (which, if I understand correctly is supposed to cause a LaTeX error because it is undefined) by the \PackageError command as shown below.

Reply:
I modified the field contents but the idea to use `\PackageError` is very nice I think :)

Reply:
(The first entry is the displayed text, so it shouldn't contain an unknown command `\quantumbsterror`, the second entry is the help text, in which I simply refer to the demo.)